integreat_cms/static/src/js/media-management/component/edit-directory-sidebar.tsx

```diff
@@ -124,6 +126,7 @@ const EditDirectorySidebar = ({
                                     }`}
                                     className={cn("btn", {
                                         "btn-red": !isLoading && directory.numberOfEntries === 0,
+                                        "hidden": !canDeleteDirectory,
```

Comment:
Normally, in (p)react, you don't add the class "hidden" to elements which should not be displayed, but you just don't render it, see e.g. [Conditional Rendering](https://reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator):

```
{canDeleteDirectory && (
    <button
        ...
    </button>
)}
```

Reply:
That's nice to know 😃 Thank you for the advice! Now applied.